designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ScenarioParametersApiSpec.scala

```diff
@@ -0,0 +1,93 @@
+package pl.touk.nussknacker.ui.api
+
+import io.restassured.RestAssured.`given`
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.test.{
+  NuRestAssureExtensions,
+  NuRestAssureMatchers,
+  PatientScalaFutures,
+  RestAssuredVerboseLogging
+}
+import pl.touk.nussknacker.ui.api.helpers.{NuItTest, NuTestScenarioManager}
+
+class ScenarioParametersApiSpec
+    extends AnyFreeSpecLike
+    with NuItTest
+    with NuTestScenarioManager
+    with NuRestAssureExtensions
+    with NuRestAssureMatchers
+    with RestAssuredVerboseLogging
+    with PatientScalaFutures {
+
+  "The endpoint for getting scenario parameters combination when" - {
+    "authenticated with write access to all categories should" - {
+      "return scenario parameters combination for all categories" in {
+        given()
+          .basicAuth("admin", "admin")
+          .when()
+          .get(s"$nuDesignerHttpAddress/api/scenarioParametersCombinations")
+          .Then()
+          .statusCode(200)
+          .equalsJsonBody(s"""
```

Comment:
What exactly do you mean? Should I remove on empty blank line in the body? Or you would like to add a new line before the string? The second option is impossible because scalafmt corrects this.

Reply:
Why is it impossible? You can check: pl.touk.nussknacker.ui.api.AppApiSpec or pl.touk.nussknacker.ui.api.ScenarioActivityApiSpec for examples

Reply:
I don't know why :) I add a new line, do reformat and it back to the previous state. Check by yourself and tell me how to deal with this :)

Reply:
I formatted like this:
```
          .equalsJsonBody(
            s"""
               |[
               |    {
               |        "processingMode": "Unbounded-Stream",
               |        "category": "Category1",
               |        "engineSetup": {
               |            "name": "Flink",
               |            "errors": [
               |
               |            ]
               |        }
               |    },
               |    {
               |        "processingMode": "Unbounded-Stream",
               |        "category": "Category2",
               |        "engineSetup": {
               |            "name": "Flink",
               |            "errors": [
               |
               |            ]
               |        }
               |    }
               |]""".stripMargin
          )
```
and then I commited changes. Scalafmt does not change anything.